src/azure.ad/basicauth_inbound_handler.bal

```diff
@@ -87,11 +88,11 @@ public type InboundUserAuthenticatorProvider object {
             passwordGrantConfig.clientSecret = clientSecret;
         }
 
-        http:BearerAuthHandler bearerAuthHandler = getOutboundOAuth2BearerHandler(passwordGrantConfig);
-        auth:OutboundAuthProvider provider = <auth:OutboundAuthProvider>bearerAuthHandler.authProvider;
+        oauth2:OutboundOAuth2Provider oAuthProvider = getOutboundOAuth2Provider(passwordGrantConfig);
```

Comment:
```suggestion
        oauth2:OutboundOAuth2Provider oauth2Provider = getOutboundOAuth2Provider(passwordGrantConfig);
```

Reply:
I have fixed this with https://github.com/ballerina-platform/module-ballerinax-azure.ad/pull/5/commits/fcf2013c0a9b2ff857af7d5791ce11ee566c980c